source/tools/testConsistency/ClientApp/app.js

```diff
@@ -0,0 +1,120 @@
+ // Copyright (c) Microsoft Corporation. All rights reserved. 
+ // Licensed under the MIT License. 
+const express = require("express");
```

Comment:
Are we running this through Semmle automatically at this point? @paulcam206?

Reply:
If we not we should run linters manually through the code base to ensure we are clean/kosher.

---
In reply to: [453430695](https://github.com/microsoft/AdaptiveCards/pull/4326#discussion_r453430695) [](ancestors = 453430695)